docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,154 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives stream of POJOs as input and using JDBC connection will insert the POJOs in a database table where each POJO corresponds to a row in the database table. The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. To execute/fire the insert queries on the database table, this operator uses PreparedStatement, a precompiled SQL statement.
```

Comment:
@amberarrow  thank you for the inputs. I have modified the content please have a look at it and let me know your thoughts. I am not elaborating on "as to what happens if a row for a tuple already exists" because we are not handling it any special way (we are not making any use of upsert queries), if the underlying table allows duplicate rows it will be added else it will fail with an SQLException. We are claiming that operator will ignore replayed tuples in case of auto recovery.